users_ldap_groups/__openerp__.py

```diff
@@ -27,28 +27,6 @@
     "license": "AGPL-3",
     "description": """
```

Comment:
replace `description` by `summary` to have the following sentence show up beneath the module's name in kanban view